react-validated-input-component/my-app/src/ValidatedInput.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+import './ValidatedInput.css';
+export default function ValidatedInput() {
+  let [count, setCount] = useState(0);
+
+  function typeCount(event) {
+    setCount((count = event.target.value.length));
```

Comment:
This is not the right way to set the count. Your state variables should always be `const`. Just set the state variable and let React's re-render logic do the rest for you.
(`setCount(event.target.value.length)`)